src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Tov.cpp

```diff
@@ -177,11 +177,9 @@ void TovSolution::integrate(
           central_specific_internal_energy, central_pressure)));
   lindblom_rhs<CoordSystem>(make_not_null(&dvars), vars, central_log_enthalpy,
                             equation_of_state);
-  double initial_step =
-      -std::min(std::abs(1.0 / dvars[0]), std::abs(1.0 / dvars[1]));
-  if constexpr (CoordSystem == TovCoordinates::Isotropic) {
-    initial_step = -std::min(std::abs(initial_step), std::abs(1.0 / dvars[2]));
-  }
+  const double initial_step =
```

Comment:
Can you please document why you made this change and the motivation? The previous definition is from Hairer and is what is canonically recommended as the initial step size. Yours is quite arbitrary as far as I can tell.

Reply:
I dropped this change and instead tightened the absolute tolerance. I'm a bit worried about the tolerances here.. they seem rather fragile. An absolute tolerance of 1e-14 wasn't enough for the XCTS constraints. Also, loosening the relative tolerance to just 1e-12 fails the `test_tov_dp_dr`. And, when tightening the absolute tolerance further (to, say, 1e-30 or even 0), the `test_tov` times out in the Newtonian limit case.